buildscripts/semantic-convention/generate.sh

```diff
@@ -4,7 +4,7 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 ROOT_DIR="${SCRIPT_DIR}/../../"
 
 # freeze the spec & generator tools versions to make SemanticAttributes generation reproducible
-SEMCONV_VERSION=1.19.0
+SEMCONV_VERSION=1.19.1
```

Comment:
This field refers to a particular [release of the opentelemetry-specification](https://github.com/open-telemetry/opentelemetry-specification/releases) repo. There is no `1.19.1` release, so running `./buildscripts/semantic-convention/generate.sh` fails. I think you just mean to add a comment rather than updating the version, so updating this field is unnecessary

Reply:
Thanks for the quick feedback. I've changed the PR accordingly.